proto/decentraland/social/friendships/friendships.proto

```diff
@@ -89,6 +89,35 @@ message Payload {
   optional string synapse_token = 1;
 }
 
+message Positions {
+  repeated Position positions = 1;
+}
+
+message Position {
+  optional string realm = 1;
```

Comment:
I'm not sure what we do with the realm, but I think it would be a good idea to check it and consider if it'd be better to change the type from string to enum

Reply:
I’m not sure also how does the explorer handle it. Anyway, when being in a world the world name is the realm, so at first we can not list all the possible names.